pyacq/devices/eeg_emotiv.py

```diff
@@ -140,9 +137,14 @@ def __init__(self, **kargs):
         self.device_info = dict()
 
     def _configure(self, device_handle='/dev/hidraw0'):
-        
+        '''
+        Parameters
+        ----------
+        device_handle : str
+            Path to the usb hidraw used
```

Comment:
Would be helpful to have something like `(for example, '/dev/hidraw0' on linux or '.....' on windows)`